### PR TITLE
[WIP] Review for Scrapy Project for SDS

### PR DESCRIPTION
**Pull request by junaidikhlaq**

### CODE REVIEW

File: sds_scraper/sds_scraper/spiders/Ivoclar_Vivadent_AG.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+class IvoclarVivadent(CrawlSpider):
+    name = 'ivoclar_vivadent'
+    base_url = 'https://www.ivoclarvivadent.se/'
+
+    def start_requests(self):
+        url = 'https://www.ivoclarvivadent.se/sv/skerhetsdatablad/'
+
+        yield scrapy.Request(url=url, callback=self.parse)
```
Comment (wadoodislam) [inline sds_scraper/sds_scraper/spiders/Ivoclar_Vivadent_AG.py]: no need for this method use `start_urls` field.

File: sds_scraper/sds_scraper/spiders/Ivoclar_Vivadent_AG.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    def parse(self, response):
+        # 1769 files
+        for items in response.xpath(
+                '//div[@class="grid__item one-half palm--one-whole"]/div[@class="item"]/div[@class="text"]').getall():
```
Comment (wadoodislam) [inline sds_scraper/sds_scraper/spiders/Ivoclar_Vivadent_AG.py]: try converting this xpath into equivalent css and use that, also its too long a path optimize it.

**Review by wadoodislam — CHANGES_REQUESTED** [review]

File: sds_scraper/sds_scraper/spiders/Ivoclar_Vivadent_AG.py (hunk 1 above)
Comment (junaidikhlaq) [reply sds_scraper/sds_scraper/spiders/Ivoclar_Vivadent_AG.py]: When I use start_urls it doesn't scrap any data

**Review by wadoodislam — APPROVED** [review]
LGTM

**Pull request merged**